sklearn/linear_model/logistic.py

```diff
@@ -233,7 +227,8 @@ def _multinomial_loss(w, X, Y, alpha, sample_weight):
 
     Parameters
     ----------
-    w : ndarray, shape (n_classes * n_features,) or (n_classes * (n_features + 1),)
+    w : ndarray,
+        shape (n_classes * n_features,) or (n_classes * (n_features + 1),)
```

Comment:
>
> It doesn't as this is a private function.
>
ok but would you recommend to split the docstring line like this?
in the old days :) docstrings were where we allowed pep8 violations of long
lines....